SAXPY tutorial: roc-obj replace with llvm-objdump (#3835)
ROCm-Developer-Tools/HIP: docs/tutorial/saxpy.rst

        s_waitcnt vmcnt(0)                                         // 000000001970: BF8C0F70
        v_fmac_f32_e32 v5, s0, v4                                  // 000000001974: 760A0800
        global_store_dword v[0:1], v5, off                         // 000000001978: DC708000 007F0500
        s_endpgm                                                   // 000000001980: BF810000
        s_nop 0                                                    // 000000001984: BF800000

    Alternatively, call the compiler with ``--save-temps`` to dump all device
    binary to disk in separate files.

    .. code-block:: bash

      amdclang++ ./HIP-Basic/saxpy/main.hip -o saxpy -I ./Common -lamdhip64 -L /opt/rocm/lib -O2 --save-temps --offload-arch=gfx942

    List all the temporaries created while compiling ``main.hip`` with:

    .. code-block:: bash

      ls main-hip-amdgcn-amd-amdhsa-*
      main-hip-amdgcn-amd-amdhsa-gfx942.bc
      main-hip-amdgcn-amd-amdhsa-gfx942.o
      main-hip-amdgcn-amd-amdhsa-gfx942.out.resolution.txt
      main-hip-amdgcn-amd-amdhsa-gfx942.hipi 
      main-hip-amdgcn-amd-amdhsa-gfx942.out
      main-hip-amdgcn-amd-amdhsa-gfx942.s

>     

Reviewer: ```suggestion

```

    Files with the ``.s`` extension hold the disassembled contents of the binary.
    The filename notes the graphics IPs used by the compiler. The contents of

>     this file are similar to what ``llvm-objdump`` printed to the console.

Reviewer: ```suggestion
    this file are similar to the `*.s` file created with ``llvm-objdump`` earlier.
```
